subsys/testsuite/include/zephyr/interrupt_util.h

```diff
@@ -165,12 +165,21 @@ static inline void trigger_irq(int irq)
 }
 
 #elif defined(CONFIG_RISCV)
+#if defined(CONFIG_SOC_SERIES_RP2350)
```

Comment:
Maybe this (and similar places) should use `CONFIG_HAZARD3_INTC`? Hazard3 is a generic CPU core that is likely to be implemented in further RP2 chips, as well as other designs (e.g. FPGAs or other vendors' SoCs)